Sources/Starknet/Data/StorageProof/NodeHashToNodeMappingItem.swift

```diff
@@ -0,0 +1,11 @@
+public typealias NodeHashToNodeMapping = [StarknetNodeHashToNodeMappingItem]
```

Comment:
```suggestion
public typealias StarknetNodeHashToNodeMapping = [StarknetNodeHashToNodeMappingItem]
```
<a href="https://emoji.gg/emoji/7945-trollface"><img src="https://cdn3.emoji.gg/emojis/7945-trollface.png" width="24px" height="24px" alt="TrollFace"></a>